src/upasana/assignment_16/Frequency.java

```diff
@@ -0,0 +1,26 @@
+package upasana.assignment_16;
+
+import java.util.Scanner;
+
+public class Frequency {
+	int count=0;
```

Comment:
You don't require to create this at instance level you can use this as local variable (I mean you should define this as local variable as this is useful to only single method).